lsp-clojure.el

```diff
@@ -37,6 +37,83 @@
   :risky t
   :type 'list)
 
+;; Refactorings
+
+(defun lsp-clojure--refactoring-call (refactor-name &rest additional-args)
+  "Send an executeCommand request for REFACTOR-NAME with ADDITIONAL-ARGS if there are more arguments expected after the line and column numbers."
+  (lsp--cur-workspace-check)
+  (lsp--send-execute-command
+   refactor-name
+   (append (list (concat "file://" buffer-file-name)
```

Comment:
use lsp--path-to-uri

Reply:
... and you can use `(list*` instead of `(append (list... )  (list ..))`

Reply:
done!